src/app/containers/Figure/fixtureData.jsx

```diff
@@ -38,6 +38,7 @@ const captionBlockWithLink = {
                     model: {
                       text: 'containing an inline link',
                       locator: 'https://www.bbc.com',
+                      isExternal: false,
                       blocks: [
                         {
```

Comment:
Can we maybe add Inline link to the Simorgh Storybook now that its some unique container logic please? One external and internal would be nice

Reply:
These have been added.